app/public/src/game.js

```diff
@@ -0,0 +1,87 @@
+(function(){'use strict';}());
+
+function Game(){
+  var frames, gameOver, totalScore, allowedNoOfFrames;
```

Comment:
note these variables are superfluous as they go out of scope after each invocation of the function.  They are not related to the property assignments below.